src/createReactQueryClient.js

```diff
@@ -0,0 +1,16 @@
+import { QueryClient } from 'react-query';
+
+const cacheTimeMinutes = 5;
+
+const createReactQueryClient = () => new QueryClient({
+  // https://react-query.tanstack.com/guides/important-defaults
+  defaultOptions: {
+    queries: {
+      retry: false,
+      refetchOnWindowFocus: false,
+      cacheTime: 1000 * 60 * cacheTimeMinutes,
```

Comment:
Five seconds is the default value for the `cacheTime` option and I added it here for now to decide the value for the FOLIO. [caching article](https://react-query.tanstack.com/guides/caching) helps to understand what is the idea of the option. I think the `staleTime` option which is mentioned in the article should not be updated so I did not add it in the PR.

Reply:
How smart is react-query about punching through the cache when something is mutated? Eg, if I mutate a `loan`, will the previously fetched and cached loan be refetched or served from cache? Will the collection of `loans` that I previously fetched be refetched?

Feel free to just throw me at docs that explain this :)

Reply:
`react-query` won't be refetching by default. For that, the [query invalidation](https://react-query.tanstack.com/guides/invalidations-from-mutations) should be used. If you received the updated information from the server after some mutation, there is also the possibility to update the cache without prefetching stuff.

Reply:
Hm, it seems the query invalidation is done using the name of the query. For our use case that presents a problem as queries span apps and we have no idea what third party app might be using some endpoint we know our mutation has affected. One thing to look into would be a mechanism to invalidate queries by endpoint rather than query name and to raise an issue with the react-query team if that's not possible. Though... it should be possible to go the long way around and generate a list of query names based on an endpoint by looping through the queries.

Reply:
Jason, I think you can use the more complex forms of [Query Keys](https://react-query.tanstack.com/guides/query-keys#array-keys) that would support some sort of unique parameters.

...I suppose this does raise the question of whether we should export a `useModuleQuery()` hook that appends the module name to the query key array (or array-ifies it if it's not already).

Reply:
I think this is possible to use endpoint as the key. See the example for the [default query fn](https://react-query.tanstack.com/guides/default-query-function). 

I was more thinking about adding an app context to the beginning of the query. For example for data-export use, the key should start with ['data-export', ...other endpoint specific props].

Reply:
I also do like the fact that it makes it possible to share queries if you explicitly _want_ to do that. As we split responsibilities across modules, things like controlled vocab lists are an easy candidate for cacheing in app groups like Agreements and Licenses.

Or things like user data for metadata components.